core/src/main/scala/io/aiven/guardian/kafka/Errors.scala

```diff
@@ -6,4 +6,8 @@ object Errors {
   case object ExpectedStartOfSource extends Errors {
     override def getMessage: String = "Always expect a single element at the start of a stream"
   }
+
+  final case class UnhandledStreamCase[T](elems: Seq[T]) extends Errors {
+    override def getMessage: String = s"Unhandled case for stream ${elems.map(_.toString.mkString(","))}"
```

Comment:
Shouldn't this be:
```suggestion
    override def getMessage: String = s"Unhandled case for stream ${elems.map(_.toString).mkString(",")}"
```
Like, `toString` all elements of the Sequence and then concat with `,`

Reply:
Good pickup, thanks!